icons/solidjs/solidjs-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><defs><linearGradient id="a" x1="27.5" x2="152" y1="3" y2="63.5" gradientTransform="matrix(.8 0 0 .8 -2.97213 2.26858)" gradientUnits="userSpaceOnUse"><stop offset=".1" stop-color="#518ac8"/><stop offset=".3" stop-color="#518ac8"/><stop offset="1" stop-color="#518ac8"/></linearGradient><linearGradient id="b" x1="95.8" x2="74" y1="32.6" y2="105.2" gradientTransform="matrix(.8 0 0 .8 -2.97213 2.26858)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#518ac8"/><stop offset=".5" stop-color="#518ac8"/><stop offset="1" stop-color="#518ac8"/></linearGradient><linearGradient id="c" x1="18.4" x2="144.3" y1="64.2" y2="149.8" gradientTransform="matrix(.8 0 0 .8 -2.97213 2.26858)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#518ac8"/><stop offset=".5" stop-color="#518ac8"/><stop offset="1" stop-color="#518ac8"/></linearGradient><linearGradient id="d" x1="75.2" x2="24.4" y1="74.5" y2="260.8" gradientTransform="matrix(.8 0 0 .8 -2.97213 2.26858)" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#518ac8"/><stop offset=".5" stop-color="#518ac8"/><stop offset="1" stop-color="#518ac8"/></linearGradient></defs><path fill="#518ac8" d="M127.6114 30.308S85.15173-.93593 52.30556 6.2742l-2.40337.80113c-4.80676 1.60225-8.81239 4.00563-11.21577 7.21013l-1.60225 2.40338-12.01689 20.82927 20.82928 4.00563c8.81238 5.60789 20.02815 8.01126 30.44278 5.60789l36.8518 7.21013z"/><path fill="url(#a)" d="M127.6114 30.308S85.15173-.93593 52.30556 6.2742l-2.40337.80113c-4.80676 1.60225-8.81239 4.00563-11.21577 7.21013l-1.60225 2.40338-12.01689 20.82927 20.82928 4.00563c8.81238 5.60789 20.02815 8.01126 30.44278 5.60789l36.8518 7.21013z" opacity=".3"/><path fill="#518ac8" d="m38.68642 30.308-3.2045.80112c-13.61914 4.00563-17.62477 16.82364-10.41464 28.0394 8.01126 10.41465 24.8349 16.02253 38.45405 12.0169l49.66981-16.82365S70.73146 23.09786 38.68642 30.308z"/><path fill="url(#b)" d="m38.68642 30.308-3.2045.80112c-13.61914 4.00563-17.62477 16.82364-10.41464 28.0394 8.01126 10.41465 24.8349 16.02253 38.45405 12.0169l49.66981-16.82365S70.73146 23.09786 38.68642 30.308z" opacity=".3"/><path fill="url(#c)" d="M104.37875 66.35866a36.05067 36.05067 0 0 0-38.45404-12.01689L16.25489 70.3643.23237 98.4037l89.72612 15.2214L105.981 84.78456c3.2045-5.60788 2.40337-12.0169-1.60226-18.4259z"/><path fill="url(#d)" d="M88.35623 94.39807A36.05067 36.05067 0 0 0 49.9022 82.38118L.23237 98.4037s42.45968 32.04505 75.30585 24.03379l2.40338-.80112c13.61914-4.00564 18.4259-16.82366 10.41463-27.2383z"/></svg>
```

Comment:
Seems that this icon also is not well aligned to the center and the paths are not unified into one.
If you need some help in unifying the paths on Inkscape, take a look at this [comment](https://github.com/devicons/devicon/issues/1306#issuecomment-1296052892).
And if you want some help about alignment on Inkscape, just keep reading.

### Align to the center of the page
> **Steps to follow if you're editing some `original` or `original-wordmark` version that has more than one path will be marked with `*`**
- Open the "Align and Distribute" tab by going to `Object > Align and Distribute...` or by key command `Shift + Ctrl + A`
- \* Select all `Ctrl + A` and group it `Ctrl + G`
- On `Align`, select `Relative to: Page`
- Select to center to the vertical and horizontal axis
- \* Select all `Ctrl + A` and ungroup it `Shift + Ctrl + G`